homeassistant-supervised/DEBIAN/control

```diff
@@ -3,7 +3,7 @@ Section: base
 Version: 1.0.2
 Priority: optional
 Architecture: all
-Depends: curl, bash, docker-ce, dbus, network-manager, apparmor, jq, systemd, os-agent
+Depends: curl, bash, docker-ce | docker.io, dbus, network-manager, apparmor, jq, systemd, os-agent
```

Comment:
This is old and should not be used and not be added here.
<https://docs.docker.com/engine/install/debian/#uninstall-old-versions>

Reply:
I agree @ludeeus